contracts/p1/BasketHandler.sol

```diff
@@ -339,7 +339,14 @@ contract BasketHandlerP1 is ComponentP1, IBasketHandler {
                 : assetRegistry.toAsset(basket.erc20s[i]).price();
 
             low256 += qty.safeMul(lowP, RoundingMode.FLOOR);
-            high256 += qty.safeMul(highP, RoundingMode.CEIL);
+
```

Comment:
I'm going to assume you mean `<` here, and yeah I think that makes sense.

The only reason I set it to `!=` was because I could not see a way for it to exceed `FIX_MAX` in any possible way given the checks and functions.

Reply:
I assume you mean `<` here and yeah I think that makes sense.

Reply:
oops, yes. But yeah, you get the point.